### ISSUE TITLE
Update to be a redirect to wichacks.io

### ISSUE THREAD

**Issue by sman591** [Women-in-Computing-at-RIT/wichacks.rit.edu#1]
wichacks.io will soon be the main website, with all content in this repo being moved to the new wichacks.io repo (move in progress). **Once the new website is ready**, this repo should become just a `301 Moved Permanently` redirect to `https://wichacks.io`. So:

* wichacks.rit.edu -> wichacks.io
* wichacks.rit.edu/extras.php -> wichacks.io
* wichacks.rit.edu/foo-bar -> wichacks.io

Stretch goal: have all links to wichacks.rit.edu go to wichacks.io - eg:
* wichacks.rit.edu -> wichacks.io
* wichacks.rit.edu/extras.php -> wichacks.io/extras
* wichacks.rit.edu/foo-bar -> wichacks.io
